lib/Zonemaster/CLI.pm

```diff
@@ -312,7 +312,7 @@ has 'elapsed' => (
     isa           => 'Bool',
     required      => 0,
     default       => 0,
-    documentation => 'Print elapsed time at end of run.',
+    documentation => __( 'Print elapsed time (in seconds) at end of run.' ),
```

Comment:
This could be interpreted that it is presented at the end, which is not necessarily true for JSON output.